dask/dataframe/_accessor.py

```diff
@@ -0,0 +1,79 @@
+import warnings
+
+
+# Ported from pandas
+# https://github.com/pandas-dev/pandas/blob/master/pandas/core/accessor.py
+
+class CachedAccessor:
```

Comment:
Python 2 test were failing because `CachedAccessor` didn't inherit from `object` explicitly, causing the

```python
object.__setattr__(obj, self._name, accessor_obj)
```

line to not work properly. 

```suggestion
class CachedAccessor(object):
```

Reply:
Ah, thanks. I copied this from pandas, which removed all those.